README.md

```diff
@@ -105,6 +105,30 @@ bash .scripts/cleanBuildAndRun.sh ios
 bash .scripts/cleanBuildAndRun.sh both
 ```
 
+### Launching the Demo App for Apple TV and Android TV
+
+Assuming a [react-native environment][react-native-environment] is set up, from the root of the repository:
+
+```bash
+npm install
+npm run bootstrap
+cd DemoAppTV
+yarn
+```
+
+**For Android TV:**
+
+```bash
+yarn android
+```
+
+_Note_: Linux users who want to run the DemoApp for Android, would also need to run `yarn start` in a separate terminal.
+
+**For tvOS:**
+
+```bash
+yarn pods && yarn tvos
```

Comment:
I don't think `yarn pods` work. Am I missing something?
I needed to launch `pod update` from the `ios` subfolder.

Also `yarn tvos` launches `xcodebuild` with the parameters about the simulator `Apple TV`. In my case the name is different `Apple TV (15.2)` probably because I installed the simulator later. I don't know if there is a better way to select the simulator. However, I'm not sure that's the problem because I've changed the name in the `package.json` and relaunched `yarn tvos` and then it complained that `'DemoAppTV.xcworkspace' does not exist.`. This because we need to switch to the subfolder where it can find the workspace.

Let me know if I just missed a step somewhere and I'm in the wrong track!

Reply:
Sorry, I must have missed adding the `pods` script to the package.json for the demo app. I have just added it there, can you please try again with the `yarn pods` command and `yarn tvos` after it? I think that should help with the missing `DemoAppTV.xcworkspace`.

Not sure how to solve the different simulator name – if I don't put the simulator name there, it will launch an iPhone simulator by default, so I have to put there something. I added a note to the README to update the package.json in case of different simulator.

Also added a note to README to update the collector endpoint in `DemoAppTV/App.js`.